File: 6.productorsum.c
#include<stdio.h>  
#include <string.h>

int main()  
{
    int number;  
    char operation[40];  
    
    printf("Enter Number : "); 
    scanf("%d",&number);  
    printf("Enter operation, product or sum: "); 
    scanf("%s",&operation);

   int sum=0;
    
    if(strcmp(operation,"sum")==0){
        for(int i = 1; i <= number; i++){ 
            sum = sum + i; 
        } 
    }
    if(strcmp(operation,"product")==0){
        sum=1;
        for(int i = 2; i <= number; i++){
            sum *= i;
        }
    }

   printf("Result: %d", sum);  
}

File: 6.productorsum.cs
using System;

class Program{
    static void Main(){
      Console.WriteLine("Enter number: ");
      int number = Convert.ToInt32(Console.ReadLine());
      Console.WriteLine("Enter operation, product or sum: ");
      string operation = Console.ReadLine();
      int sum = 0;
      if(operation == "sum"){
        sum = number*(number+1)/2;
      }
      if(operation == "product"){
        sum=1;
        for(int i = 2; i <= number; i++){
            sum *= i;
        }
      }
      Console.WriteLine("Result: " + sum); 
    }
}